src/Kunstmaan/NodeSearchBundle/Resources/doc/ElasticSearch.md

```diff
@@ -0,0 +1,50 @@
+# Elastic search support
+
+We have added support for Elastic search up to version 6.x.
+When you have a project that is using PHP 7 then you can use version 6.x when the project is using ruflin/elastica >= 6.0.
+
+## Dynamic configuration
+We have added dynamic support to detect which version your project is using.
+Dependancies for using the configuration of version 6 is that your project uses PHP 7, 
```

Comment:
Typo, can you check the document again for typo's ? 😄

Reply:
I have updated the doc. and removed the typo :)